interfaces/acados_matlab/acados_ocp.m

```diff
@@ -102,8 +99,13 @@ function set(varargin)
 
 
 		function delete(obj)
-			ocp_destroy_ext_fun(obj.model_struct, obj.C_ocp, obj.C_ocp_ext_fun);
-			ocp_destroy(obj.C_ocp);
+      if ~isempty(obj.C_ocp_ext_fun)
```

Comment:
How can this be empty? Both are always created in the constructor, right?

Reply:
The compilation is done in the constructor. When you git `ctrl+c` during compilation, or if there is any error in the constructor, still the destructor is being called (at least on windows) which leads to a mex crash because the functions e.g. `ocp_destroy_ext_fun` have invalid arguments.